### PR TITLE
fix(types): Add definition of returning in SaveOptions as it was missing.

### PR DESCRIPTION
**Pull request by onozaty**
## Pull Request Checklist

- [ ] ~~Have you added new tests to prevent regressions?~~
- [ ] ~~If a documentation update is necessary, have you opened a PR to [the documentation repository]~~(https://github.com/sequelize/website/)? 
- [ ] ~~Did you update the typescript typings accordingly (if applicable)?~~
- [x] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Does the name of your PR follow [our conventions](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md#6-commit-your-modifications)?

## Description Of Change

In the type definition file, the return of SaveOptions was omitted. 

https://github.com/sequelize/sequelize/blob/cb8ea88c9aa37b14c908fd34dff1afc603de2ea7/src/model.d.ts#L1284-L1307

They are present in the actual code.

https://github.com/sequelize/sequelize/blob/cb8ea88c9aa37b14c908fd34dff1afc603de2ea7/src/model.js#L4016-L4022

This pull request adds the missing `returning`.

There was no problem with the v7 codes.

### CODE REVIEW

File: src/model.d.ts [hunk 1]
```diff
@@ -1304,6 +1304,11 @@ export interface SaveOptions<TAttributes = any> extends Logging, Transactionable
    * @default false
    */
   omitNull?: boolean;
+
+  /**
+   * Return the affected rows (only for postgres)
+   */
+  returning?: boolean | (keyof TAttributes)[];
```
Comment (ephys) [inline src/model.d.ts:1311]:
```suggestion
  returning?: boolean | Array<keyof TAttributes>;
```

**Review by ephys — APPROVED** [review]
Small code style change, but otherwise good to go

File: src/model.d.ts (hunk 1 above)
Comment (onozaty) [reply src/model.d.ts]: Thanks.

**Review by ephys — APPROVED** [review]
Thank you!

**Pull request merged**